truffle/src/main/java/org/jruby/truffle/RubyFiletypeDetector.java

```diff
@@ -0,0 +1,50 @@
+package org.jruby.truffle;
```

Comment:
Yes org.jruby might be right. And the metadata would then need to go in the core jar obviously.